src/ImageSharp/ColorProfiles/ColorProfileConverterExtensionsIcc.cs

```diff
@@ -189,6 +189,9 @@ private static Vector4 GetTargetPcsWithoutAdjustment(
                 sourcePcs = sourceParams.Is16BitLutEntry ? LabV2ToLab(sourcePcs) : sourcePcs;
                 CieLab lab = CieLab.FromScaledVector4(sourcePcs);
                 CieXyz xyz = pcsConverter.Convert<CieLab, CieXyz>(in lab);
+
+                // DemoMaxICC clips negatives as part of IccUtil.cpp : icLabToXYZ > icICubeth
+                xyz = new CieXyz(Vector3.Clamp(xyz.ToVector3(), Vector3.Zero, new Vector3(float.MaxValue, float.MaxValue, float.MaxValue)));
```

Comment:
The fiddly edge cases are cropping up. Is there a simpler way to do this negative clipping?

Reply:
```suggestion
                xyz = new CieXyz(Vector3.Max(xyz.ToVector3(), Vector3.Zero));
```
you shouldn't need the float.Max part as it has to already be less or equal to max to exist in the first place, so just a Vector based max

Reply:
Doh!